resources/uploads/models/upload_updatable.yml

```diff
@@ -11,4 +11,12 @@ properties:
     type: boolean
     description: >
       Properties in `columnMapping` will be appended to the existing
-      `columnMapping` object if set to `false`. If set to `true`, the existing `columnMapping` object will be overwritten with the data supplied in `columnMapping` property.
+      `columnMapping` object if set to `false`. If set to `true`, the existing `columnMapping` object will be overwritten with the data supplied in `columnMapping` property. <br/>Please note that this field will be **deprecated on 31st October, 2022**.
```

Comment:
Again, we're not deprecating on 31st October, we want to show this as deprecated now.  Deprecating means "we will be removing this soon" and then the removal is separate from the deprecation.